From 8b3f2f72f20cdb028c0e32e0dc129bb1fab4bcb3 Mon Sep 17 00:00:00 2001
From: Piotr Maziarz <piotrx.maziarz@intel.com>
Date: Wed, 15 May 2019 13:19:22 +0200
Subject: [PATCH] ASoC: Intel: Skylake: Lower log level for FW notifications

Changed log level to debug since this information is used for this
purpose.
Printing Params informational may be confusing for user.

Change-Id: Ic2049e036a3d986c3258faeec2b558c1bf4b6032
Signed-off-by: Piotr Maziarz <piotrx.maziarz@intel.com>
---
 sound/soc/intel/skylake/skl-sst-ipc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/skl-sst-ipc.c b/sound/soc/intel/skylake/skl-sst-ipc.c
index 9050455..0dc6231 100644
--- a/sound/soc/intel/skylake/skl-sst-ipc.c
+++ b/sound/soc/intel/skylake/skl-sst-ipc.c
@@ -609,7 +609,7 @@ static int skl_process_module_notification(struct skl_sst *skl)
 		break;
 	}
 
-	print_hex_dump(KERN_DEBUG, "Params:",
+	print_hex_dump_debug("Params:",
 			DUMP_PREFIX_OFFSET, 8, 4,
 			&notify, sizeof(struct skl_event_notify), false);
 }
-- 
1.9.1

